src/sass_types/boolean.cpp

```diff
@@ -15,59 +13,63 @@ namespace SassTypes
     return v ? instance_true : instance_false;
   }
 
-  Handle<Function> Boolean::get_constructor() {
+  v8::Local<v8::Function> Boolean::get_constructor() {
+    Nan::EscapableHandleScope scope;
+    v8::Local<v8::Function> conslocal; 
     if (constructor.IsEmpty()) {
-      Local<FunctionTemplate> tpl = NanNew<FunctionTemplate>(New);
+      v8::Local<v8::FunctionTemplate> tpl = Nan::New<v8::FunctionTemplate>(New);
 
-      tpl->SetClassName(NanNew("SassBoolean"));
+      tpl->SetClassName(Nan::New("SassBoolean").ToLocalChecked());
       tpl->InstanceTemplate()->SetInternalFieldCount(1);
-      tpl->PrototypeTemplate()->Set(NanNew("getValue"), NanNew<FunctionTemplate>(GetValue)->GetFunction());
+      Nan::SetPrototypeTemplate(tpl, "getValue", Nan::GetFunction(Nan::New<v8::FunctionTemplate>(GetValue)).ToLocalChecked());
 
-      NanAssignPersistent(constructor, tpl->GetFunction());
+      constructor.Reset(Nan::GetFunction(tpl).ToLocalChecked());
+      conslocal = Nan::New(constructor);
 
-      NanAssignPersistent(get_singleton(false).js_object, NanNew(constructor)->NewInstance());
-      NanSetInternalFieldPointer(NanNew(get_singleton(false).js_object), 0, &get_singleton(false));
-      NanNew(constructor)->Set(NanNew("FALSE"), NanNew(get_singleton(false).js_object));
+      get_singleton(false).js_object.Reset(conslocal->NewInstance());
```

Comment:
It might be that this should be `get_singleton(false).js_object.Reset(Nan::NewInstance(conslocal));`.